docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -5,17 +5,19 @@ description: Dagster supports data backfills for each partition or subsets of pa
 
 # Backfills
 
-## Overview
+Backfilling is the process of running partitions for assets or ops that either don’t exist or updating existing records. Dagster supports backfills for each partition or a subset of partitions.
 
-Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can launch a _backfill_ which submits runs to fill in multiple partitions at the same time.
+After defining a [partition](/concepts/partitions-schedules-sensors/partitions), you can launch a backfill that will submit runs to fill in multiple partitions at the same time.
+
+Backfills are common when setting up a pipeline for the first time. The assets you want to materialize might have historical data that needs to be materialized to get the assets up to date. Another common reason to run a backfill is when you’ve changed the logic for an asset and need to update historical data with the new logic.
 
 ---
 
-## Launching Backfills
+## Launching backfills for partitioned assets
 
-### Partitioned Asset Backfills
+To launch backfills for a partitioned asset, click the **Materialize** button on either the [**Asset details**](/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets) or the **Global asset lineage** page. The backfill modal will display.
 
-You can open the backfill modal to launch backfills for a partitioned asset using the "Materialize" button, either on the [Asset detail page](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets) or when viewing a graph of assets. Backfills can also be launched for a selection of differently partitioned assets, as long as the roots share the same partitioning.
+Backfills can also be launched for a selection of partitioned assets as long as the most upstream assets share the same partitioning. For example: All partitions use a `DailyPartitionsDefinition`.
```

Comment:
```suggestion
Backfills can also be launched for a selection of partitioned assets as long as the most upstream assets share the same partitioning. For example: All assets with no upstream assets in the selection use a `DailyPartitionsDefinition`.
```

if all assets use a DailyPartitionsDefinition, that would also be a correct example, but it's the most simple case so i think using the slightly more complicated version might help illustrate the point better? i'm not really sure it's actually that much better